azure-pipelines/build.yml

```diff
@@ -59,16 +59,16 @@ steps:
     useGlobalJson: true
 
 - task: PowerShell@2
-  displayName: 'Add version number to SqlCore project for packaging'
+  displayName: 'Add version number to SqlCore nuspec for packaging'
   inputs:
     targetType: 'inline'
     script: |
-      $projectFile = '$(Build.SourcesDirectory)\src\Microsoft.SqlTools.SqlCore\Microsoft.SqlTools.SqlCore.csproj'
+      $nuspecFile = '$(Build.SourcesDirectory)\src\Microsoft.SqlTools.SqlCore\Microsoft.SqlTools.SqlCore.nuspec'
```

Comment:
FYI - you can also just pass in properties directly 

https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-using-a-nuspec-file